service/http.redirect/src/main/java/org/kaazing/gateway/service/http/redirect/HttpRedirectServiceHandler.java

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kaazing.gateway.service.http.redirect;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
+
+import org.kaazing.gateway.transport.IoHandlerAdapter;
+import org.kaazing.gateway.transport.http.HttpAcceptSession;
+import org.kaazing.gateway.transport.http.HttpMethod;
+import org.kaazing.gateway.transport.http.HttpStatus;
+import org.slf4j.Logger;
+
+class HttpRedirectServiceHandler extends IoHandlerAdapter<HttpAcceptSession> {
+    private final Logger logger;
+    private String location;
+    private HttpStatus statusCode;
+    private String cacheControl;
+
+    private static final DateFormat RFC822_FORMAT_PATTERN =
+            new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz", Locale.ENGLISH);
+    static {
+        RFC822_FORMAT_PATTERN.setTimeZone(TimeZone.getTimeZone("GMT"));
+    }
+
+    HttpRedirectServiceHandler(Logger logger) {
+        this.logger = logger;
+    }
+
+    @Override
+    public void doSessionCreated(HttpAcceptSession session) throws Exception {
+        // NOOP no license check needed
```

Comment:
I don't think  `no license check needed` are needed, is this from copy paste maybe.

Nit: Also, I don't think there is great value in having the `@Override NOOP` method, though I'm ok with them.  I don't want to dig into the API, but perhaps there is a more complete Adapter to use that doesn't require overriding them